SonarOnPull/Program.cs

```diff
@@ -10,6 +10,33 @@ class Program
     {
         static void Main(string[] args)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "args". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://138.68.127.131:9000/coding_rules#rule_key=csharpsquid%3AS1172)